src/main/scala/com/cognite/sdk/scala/v1/relationships.scala

```diff
@@ -66,17 +65,16 @@ final case class RelationshipCreate(
 ) extends WithRequiredExternalId
 
 final case class RelationshipUpdate(
-    externalId: Option[Setter[String]] = None,
     sourceExternalId: Option[NonNullableSetter[String]] = None,
     sourceType: Option[NonNullableSetter[String]] = None,
     targetExternalId: Option[NonNullableSetter[String]] = None,
     targetType: Option[NonNullableSetter[String]] = None,
-    startTime: Option[NonNullableSetter[Instant]] = None,
-    endTime: Option[NonNullableSetter[Instant]] = None,
-    confidence: Option[NonNullableSetter[Double]] = None,
-    dataSetId: Option[NonNullableSetter[Long]] = None,
+    startTime: Option[Setter[Instant]] = None,
+    endTime: Option[Setter[Instant]] = None,
+    confidence: Option[Setter[Double]] = None,
+    dataSetId: Option[Setter[Long]] = None,
     labels: Option[NonNullableSetter[Seq[CogniteExternalId]]] = None
-) extends WithSetExternalId
```

Comment:
this may create difficulties with using existing Upsert/Update utility methods so i am not sure if this is a good idea.